index.js

```diff
@@ -9,9 +9,9 @@ module.exports = (css, settings) => {
       `/*%%styled-jsx-expression-${id}%%*/`
     )
 
-  const preprocessed = sass({
+  const preprocessed = sass.renderSync(Object.assign({
     data: cssWithPlaceholders
-  }).css.toString()
+  }, settings.sassOptions)).css.toString()
```

Comment:
what happens when `settings.sassOptions` is `undefined`?

Reply:
@giuseppeg if it is `undefined` the result will be `{ data: cssWithPlaceholders }`.

Reply:
oh sorry I quick scanned it and I thought it was a 2nd param that you are passing to `renderSync` rather than `Object.assign` :)

If you rebase with master and solve the conflicts I'll merge

Reply:
can you add `node-sass` as dev dependency?

Reply:
@giuseppeg 👍 no prob, done